### ISSUE TITLE
Information on hyperparameters used when evaluating DeepForest

### ISSUE THREAD

**Issue by ethanwhite** [jonathanventura/urban-tree-detection#3]
Hi - I'm one of the deepforest developers and we have a [user working with NAIP data(https://github.com/weecology/DeepForest/discussions/807) and I was wondering if you'd find sharing the hyperparameter values you found to be most effective in <https://doi.org/10.1016/j.jag.2024.103848>.

Also - I'd just like to say that I really appreciate the conscientious approach you took to evaluating deepforest in that paper. If everyone worked like this when developing new approaches we'd make really rapid progress as a field. Thank you!

**Comment by jonathanventura** [comment]
Hi Ethan, thanks for your kind comments and for mentioning my work in the DeepForest discussion.   It was interesting to see how well DeepForest could do on our dataset which only has point annotations.

Regarding the question about hyperparameters: I did not do any tuning of the patch size and iou_threhold hyperparameters and left them at the defaults.  I did fine-tune the model itself on our data, and experimented with different bounding box sizes with a side length of 9.6 m (16 pixels) being the best performer.  I also tuned the score_thresh parameter using Optuna and found the best setting to be 0.302.

I will post this information in the other discussion board as well -- I hope it is helpful.
